text/0000-scoped-slots-light-dom.md

```diff
@@ -0,0 +1,509 @@
+    <slot>Regular Slot</slot>
+</template>
+```
+
+- Mixed named slots, one for regular slot and another for scoped slot
+```html
+// x/parent.html
+<template>
+    <x-child>
+        <template slot="slotname" lwc:slot-data="slotdata">
+            <p>{slotdata.name}</p>
+        </template>
+        <template slot="slotname">
+            <p>Chocolatier</p>
+        </template>
+    </x-child>
+</template>
+
+// x/child.html
+<template lwc:render-mode="light">
+    <slot name="slotname" lwc:slot-bind={slotdata}>Scoped Slot</slot>
+    <slot name="slotname">Regular Slot</slot>
+</template>
+```
+
+### Access to bindings
+Scoped slots have access to component bindings and scope bindings
+#### Component Bindings
+```html
+<template>
+    {title}
+    <x-list>
+        <template lwc:slot-data="item">
+            <div>{label}</div> <!-- label is a component binding and will be repeated in every row of the list-->
+            <span>{item.id} - {item.name}</span>
+        </template>
+    </x-list>
+</template>
+```
+
+#### Nested Scoped Slots
+Scoped slots that are nested within other scoped slots bring in new levels of flexibility for component authors.
+```html
+<template>
+    <x-table data={data}>
+        <template lwc:slot-data="row">
+            <x-row row={row}> <!-- this is rendered for every row in the table -->
+                <template lwc:slot-data="column">
+                    <span> <!-- this is rendered for every column in the row -->
+                        Coordinates: {row.number} - {column.number} <!-- can refer to both `row` and `column` -->
+                    </span>
+                </template>
+            </x-row>
+        <template>
+    </x-table> 
+</template>
+```
+
+### Ownership and styling implication
+Standard slots and scoped slots are identical with respect to ownership. The slotted content is owned by the parent component. This has direct implications in terms of styling. If the parent component has a scoped stylesheet associated with it, the styles also apply to the scoped slot content.
+
+### Validations
+Validation of scoped slot usage between parent and component is handled at runtime.
+* **Missing default slot**: If a parent component slots content in to the child component's default slot and the child does not have a default slot, the content will be ignored.(TODO: Why don't we log an error in this case today for standard slots?)
+* **Missing named slot**: If a parent component tries to set an unknown named slot of child component, the content will be ignored and an error will be logged in dev mode.
+
+_The above behaviors are consistent with how the engine handles regular slot content._
+
+* **Parent uses scoped slot but child has regular slot**: In this case the content will be ignored and an error will be logged in dev mode.
+* **Parent uses regular slot but child has scoped slot**: This is valid usage. The content provided by the parent will be slotted into the child's scoped slots.

```

Comment:
Just to confirm it is the following use case where `<x-list>` has a default scoped slot?

```html
<template>
    <x-child>
        <p>Chocolatier</p>
    </x-child>
</template>
```

How would this work if the child iterates over the default slot, does it means that the `<p>` will be copied multiple time?